packages/lib-user/src/translations/i18n.js

```diff
@@ -0,0 +1,39 @@
+import i18n from 'i18next'
+import {
+  initReactI18next,
+  Trans as BaseTrans,
+  useTranslation as useBaseTranslation
+} from 'react-i18next'
+
+const libI18n = i18n.createInstance()
```

Comment:
This is trivial, but if lib-classifier creates `classifierI18n` and lib-react-components creates `zrcI18n`, should this be less generic and include `user` or a related abbreviation?